ncm-metaconfig/src/main/metaconfig/opennebula/pan/schema.pan

```diff
@@ -0,0 +1,16 @@
+declaration template metaconfig/opennebula/schema;
+
+include 'pan/types';
+
+type aii_section = {
+    "password" : string
+    "host" : string
+    "port" ? long
+    "user" ? string
+    "pattern" ? string
+} = dict();
+
+@documentation{
+aii/opennebula.conf sections
+}
+type aii_opennebula_conf = aii_section{};
```

Comment:
I am just thinking about this now: It does mather in which order the sections are put into the file, because the aii component takes the first matching regex ? So I think this should be an array.. But I don't know to render Config-Tiny correctly then